PetaPoco/Utilities/ParametersHelper.cs

```diff
@@ -170,9 +170,8 @@ void ProcessArg(object arg)
                 else
                 {
                     var type = arg.GetType();
-                    // TODO: Include second param `paramName: nameof(args)` in thrown ArgumentException
                     if (type.IsValueType || type == typeof(string))
-                        throw new ArgumentException($"Value type or string passed as stored procedure argument: {arg}");
+                        throw new ArgumentException($"Value type or string passed as stored procedure argument: {arg}", nameof(args));
```

Comment:
Cheers, thanks for this. Added this comment while rewriting the xml docs and was trying not to let myself get distracted lol.